lib/ConstExtract/ConstExtract.cpp

```diff
@@ -703,6 +746,26 @@ void writeValue(llvm::json::OStream &JSON,
     break;
   }
 
+  case CompileTimeValue::KeyPath: {
+      auto keyPathValue = cast<KeyPathValue>(value);
+      JSON.attribute("valueKind", "KeyPath");
+      JSON.attributeObject("value", [&]() {
+        JSON.attribute("path", keyPathValue->getPath());
+        JSON.attribute("rootType", toFullyQualifiedTypeNameString(keyPathValue->getRootType()));
+        JSON.attribute("valueType", toFullyQualifiedTypeNameString(keyPathValue->getValueType()));
```

Comment:
Similar question here, as about the `ValueType` member of `KeyPathValue`.

Reply:
Removed it. We can compute it on demand if required.